nautobot_circuit_maintenance/forms.py

```diff
@@ -107,30 +103,32 @@ class CircuitMaintenanceFilterForm(BootstrapMixin, CustomFieldModelFilterFormMix
     end_time = forms.DateTimeField(label="End time before", required=False, widget=DateTimePicker())
 
 
-class CircuitMaintenanceBulkEditForm(BootstrapMixin, AddRemoveTagsForm, CustomFieldModelBulkEditFormMixin):
+class CircuitMaintenanceBulkEditForm(NautobotBulkEditForm, AddRemoveTagsForm):
     """Form for bulk editing Circuit Maintenances."""
 
     pk = forms.ModelMultipleChoiceField(queryset=CircuitMaintenance.objects.all(), widget=forms.MultipleHiddenInput)
-    status = forms.CharField(max_length=200, required=False)
+    status = forms.ChoiceField(choices=CircuitMaintenanceStatusChoices, required=False)
```

Comment:
Does this change possibly break something that folks have relied on. If so, we should make sure that we have the appropriate change documentation. Some may be using the CharField that we had previously.